src/Nethermind/Nethermind.Network/P2P/Subprotocols/Eth/V65/Eth65ProtocolHandler.cs

```diff
@@ -66,11 +68,43 @@ GetPooledTransactionsMessage getPooledTxMsg
                     Handle(getPooledTxMsg);
                     break;
                 case Eth65MessageCode.NewPooledTransactionHashes:
-                    Metrics.Eth65NewPooledTransactionHashesReceived++;
+                    NewPooledTransactionHashesMessage newPooledTxMsg =
+                        Deserialize<NewPooledTransactionHashesMessage>(message.Content);
+                    ReportIn(newPooledTxMsg);
+                    Handle(newPooledTxMsg);
                     break;
             }
         }
 
+        private void Handle(NewPooledTransactionHashesMessage msg)
+        {
+            Metrics.Eth65NewPooledTransactionHashesReceived++;
+            Stopwatch stopwatch = Stopwatch.StartNew();
+            List<Keccak> discoveredTxHashes = new();
```

Comment:
Shall we use flood controller? Or not expecting issues here? If we are getting tx's that aare already pending

Reply:
We are receiving every hash many times and it's totally fine - sending node doesn't know if we already know this tx hash. Full txs we are receiving only on request, receiving many tx hashes isn't a problem

Reply:
flood controller not needed, was only needed for eth62 geths

Reply:
we can still be flooded with hashes?